### ISSUE TITLE
Allow profile pics from Commons

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#1924]
Currently, we store the user profile pic locally. It would be nice to allow a user to choose an existing photo from Wikimedia Commons as an alternative.

**Comment by dustmop44** [comment]
Hi. I've taken a crack at this. I had to migrate a new column to user_profile. I also went ahead and added a remove profile image button which I could not find anywhere. For that feature, I had to add a controller method to user_profiles as well. This is my first issue so let me know if I did anything wrong. This is how it looks so far.
<img width="1119" alt="screenshot at mar 10 15-14-04" src="https://user-images.githubusercontent.com/35471751/54090126-32786200-4347-11e9-9552-9e8794119231.png">
